src/inmanta/ast/statements/define.py

```diff
@@ -102,10 +103,10 @@ def __init__(
         else:
             self.comment = None
 
-        self.parents = [str(x) for x in parents]
+        self.parents = parents
 
         if len(self.parents) == 0 and not (self.name == "Entity" and self.namespace.name == "std"):
-            self.parents.append("std::Entity")
+            self.parents.append(LocatableString("std::Entity", Range("__internal__", -1, -1, -1, -1), -1, namespace))
```

Comment:
perhaps we should move the Range to a constant?

Reply:
Do you mean a constant that can be imported from `inmanta.ast`?